Apps/Src/ReadTritium.c

```diff
@@ -0,0 +1,25 @@
+/* Copyright (c) 2021 UT Longhorn Racing Solar */
+
+#include "ReadTritium.h"
+
+void Task_ReadTritium(void* p_arg) {
+
+	OS_ERR err;
+
+	CANbus_Init();
+
+	while (1) {
+
+		uint8_t *message;

```

Comment:
This needs to be an array so it allocates space to actually store the values. Max size I believe is 8 bytes so it can just be `uint8_t message[8]`